src/CommunityToolkit.Maui.UnitTests/Converters/IsInRangeConverterTests.cs

```diff
@@ -0,0 +1,284 @@
+		new object?[] { (uint)073, (uint)7, (uint)73, TrueTestObject, FalseTestObject, TrueTestObject },
+		new object?[] { (uint)003, (uint)7, (uint)73, TrueTestObject, FalseTestObject, FalseTestObject },
+		new object?[] { (uint)373, (uint)7, (uint)73, TrueTestObject, FalseTestObject, FalseTestObject },
+		// System.UInt64
+		new object?[] { (ulong)037, (ulong)7, (ulong)73, TrueTestObject, FalseTestObject, TrueTestObject },
+		new object?[] { (ulong)007, (ulong)7, (ulong)73, TrueTestObject, FalseTestObject, TrueTestObject },
+		new object?[] { (ulong)073, (ulong)7, (ulong)73, TrueTestObject, FalseTestObject, TrueTestObject },
+		new object?[] { (ulong)003, (ulong)7, (ulong)73, TrueTestObject, FalseTestObject, FalseTestObject },
+		new object?[] { (ulong)373, (ulong)7, (ulong)73, TrueTestObject, FalseTestObject, FalseTestObject },
+		// System.UIntPtr
+		new object?[] { (UIntPtr)037, (UIntPtr)7, (UIntPtr)73, TrueTestObject, FalseTestObject, TrueTestObject },
+		new object?[] { (UIntPtr)007, (UIntPtr)7, (UIntPtr)73, TrueTestObject, FalseTestObject, TrueTestObject },
+		new object?[] { (UIntPtr)073, (UIntPtr)7, (UIntPtr)73, TrueTestObject, FalseTestObject, TrueTestObject },
+		new object?[] { (UIntPtr)003, (UIntPtr)7, (UIntPtr)73, TrueTestObject, FalseTestObject, FalseTestObject },
+		new object?[] { (UIntPtr)373, (UIntPtr)7, (UIntPtr)73, TrueTestObject, FalseTestObject, FalseTestObject },
+		//System.Version
+		new object?[] { new Version(07, 37, 07, 03), new Version(3, 37, 3, 7), new Version(37, 7, 3 ,37), TrueTestObject, FalseTestObject, TrueTestObject },
+		new object?[] { new Version(03, 37, 03, 07), new Version(3, 37, 3, 7), new Version(37, 7, 3, 37), TrueTestObject, FalseTestObject, TrueTestObject },
+		new object?[] { new Version(37, 07, 03, 37), new Version(3, 37, 3, 7), new Version(37, 7, 3, 37), TrueTestObject, FalseTestObject, TrueTestObject },
+		new object?[] { new Version(03, 03, 07, 07), new Version(3, 37, 3, 7), new Version(37, 7, 3, 37), TrueTestObject, FalseTestObject, FalseTestObject },
+		new object?[] { new Version(73, 73, 37, 37), new Version(3, 37, 3, 7), new Version(37, 7, 3, 37), TrueTestObject, FalseTestObject, FalseTestObject },
+	};
+
+	[Theory]
+	[InlineData(20d, "A", 'B', TrueTestObject, FalseTestObject)]
+	[InlineData(20d, 1d, 'B', TrueTestObject, FalseTestObject)]
+	[InlineData(20d, "A", 1d, TrueTestObject, FalseTestObject)]
+	public void InvalidIComparableThrowArgumentOutOfRangeException(IComparable value, IComparable comparingMinValue, IComparable comparingMaxValue, object trueObject, object falseObject)
+	{
+		IsInRangeConverter isInRangeConverter = new()
+		{
+			MinValue = comparingMinValue,
+			MaxValue = comparingMaxValue,
+			FalseObject = falseObject,
+			TrueObject = trueObject,
+		};
+
+		_ = Assert.Throws<ArgumentOutOfRangeException>(() => ((ICommunityToolkitValueConverter)isInRangeConverter).Convert(value, typeof(object), null, CultureInfo.CurrentCulture));
```

Comment:
A friendly reminder that we don't use the `_` (discard) pattern. Can remove this and the others that you added?